### PR TITLE
[Fix #3947] Fix Rails/FilePath for dstr calling Rails.root without path

### LINKED ISSUE #3947: False positive in Rails/FilePath cop
## Expected behavior

`Rails/FilePath` cop ([PR](https://github.com/bbatsov/rubocop/pull/3822)) shouldn't complain when I have the following code:

```ruby
system "rm -rf #{Rails.root.join('public', 'system')}"
```

## Actual behavior

`$ rubocop --only Rails/FilePath`

```ruby
Offenses:
lib/tasks/db.rake:52:12: C: Rails/FilePath: Please use Rails.root.join('path', 'to') instead.
    system "rm -rf #{Rails.root.join('public', 'system')}"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

## RuboCop version

```
0.47.1 (using Parser 2.3.3.1, running on ruby 2.3.3 x86_64-darwin16)
```

Thank you.

### PR DESCRIPTION
**Pull request by jspanjers**

### CODE REVIEW

File: spec/rubocop/cop/rails/file_path_spec.rb [hunk 1]
```diff
@@ -32,7 +32,16 @@
     end
   end
 
-  context 'when using Rails.root called by double quoted string' do
+  context 'when using Rails.root without path called by double quoted string' do
+    let(:source) { '"#{Rails.root}"' }
```
Comment (pocke) [inline spec/rubocop/cop/rails/file_path_spec.rb]: I think we need more test case.
The issue ( #3947 ) problem is `"rm -rf #{Rails.root.join("foo")}"`, so, at a minimum, this should be tested.

**Comment by easybills-admin** [comment]
Any news on this? 😄

**Comment by bbatsov** [comment]
Waiting for more test cases. Otherwise I'll close this PR.

**Comment by bbatsov** [comment]
Guess I'll close this. @easybills-admin (or anyone else) feel free to pick it up.

**Pull request closed without merging**